### DISCUSSION TITLE
Städteauswahl in Positionsangaben erweitern

### DISCUSSION THREAD

**Discussion by Weseler485** [OpenCCU/OpenCCU#2286] [Development]
Hallo Gemeinde,
ich möchte aus Eigeninteresse die Städteauswahl in den Positionsangaben erweitern. Sicherlich ist die Erweiterung auch für den ein oder anderen von euch interessant! Wenn uns Jens eine Vorgabe macht ab welcher Größe/"Einwohnerzahl" ein Ort groß genug für die Liste ist, werde ich eure Wünsche gerne berücksichtigen. Dazu dann bitte die Stadt und die Koordinaten nennen - ich werde es in der Liste erweitern.

Grüße
Maik2208

**Comment by jp112sdl** [comment]
2 Ideen dazu, um nicht eine statische Liste weiterzupflegen: 
- Städtename als Freitext eingeben und Koordinaten online beziehen, z.B. https://nominatim.openstreetmap.org/search?q=berlin&format=json
- Geolocation direkt aus dem Browser abrufen (https://www.w3schools.com/html/html5_geolocation.asp)

**Comment by jens-maus** [comment]
Ich wäre auch dafür (wenn man schon arbeit investiert) solche dynamischen Eingabemöglichkeiten zu schaffen. Im Grunde geht es doch nur damit das ein Nutzer seine Geolocation bestimmen lassen kann. Der Städtename wird ja AFAIK nirgends wirklich weiterverwendet. Deshalb wäre ich auch dafür nicht einfach nur die statische Liste zu vergrößern, sondern Geolocation über Browser zu ermöglichen, d.h. ein Button neben die Geolocation Eingabe zu machen die genau das macht was da bei w3schools.com beschrieben ist. Und dann der Städteeingabename einfach als Freitext zu machen damit der Nutzer sich dran erfreuen kann das sein selbst gewählter Städtename dann da steht - oder eben einfach nur ein Platzhalter dort anzuzeigen.

**Comment by Sineos** [comment]
Oder einfach eine Eingabemöglichkeit in Längen und Breitengrad und jeder kann das exakt aus den x-beliebigen online Quellen befüllen.
Das einzige wo es Sinn macht sind Astrozeitberechnungen

**Reply by jp112sdl** [reply]
> Oder einfach eine Eingabemöglichkeit in Längen und Breitengrad

@Sineos Hast du dir die "Zeit-/Position" Einstellungsseite jemals angeschaut?

**Reply by Sineos** [reply]
Heh, stimmt. Wahrscheinlich schon seit Jahren nicht mehr 😏

**Reply by jp112sdl** [reply]
Oder noch nie?
Die Eingabe von Längen- und Breitengrad gab es schon immer.
